arm/Microsoft.OperationalInsights/workspaces/readme.md

```diff
@@ -122,16 +122,16 @@ Tag names and tag values can be provided as needed. A tag can be left without a
 
 | Output Name | Type | Description |
 | :-- | :-- | :-- |
-| `logAnalyticsName` | string | The name of the deployed log analytics workspace |
-| `logAnalyticsResourceGroup` | string | The resource group of the deployed log analytics workspace |
-| `logAnalyticsResourceId` | string | The resource ID of the deployed log analytics workspace |
 | `logAnalyticsWorkspaceId` | string | The ID associated with the workspace |
+| `name` | string | The name of the deployed log analytics workspace |
+| `resourceGroupName` | string | The resource group of the deployed log analytics workspace |
+| `resourceId` | string | The resource ID of the deployed log analytics workspace |
 
 ## Template references
 
 - [Diagnosticsettings](https://docs.microsoft.com/en-us/azure/templates/Microsoft.Insights/2021-05-01-preview/diagnosticSettings)
 - [Locks](https://docs.microsoft.com/en-us/azure/templates/Microsoft.Authorization/2017-04-01/locks)
-- [Roleassignments](https://docs.microsoft.com/en-us/azure/templates/Microsoft.Authorization/2021-04-01-preview/roleAssignments)
+- [Roleassignments](https://docs.microsoft.com/en-us/azure/templates/Microsoft.Authorization/roleAssignments)
```

Comment:
Shouldn't we include the API version in the link? This will automatically redirect to latest after a new API will be out

Reply:
I know. I re-ran the generation of the readme's instead of performing changes manually. Apparently, the URL isn't actually available. A quick check confirmed this suspicion.